.github/workflows/osrm-backend.yml

```diff
@@ -85,6 +85,16 @@ jobs:
             CLANG_VERSION: 5.0.0
             CUCUMBER_TIMEOUT: 60000
 
+          - name: clang-11.0-debug-clang-tidy
+            continue-on-error: false
+            node: 12
+            runs-on: ubuntu-20.04
+            BUILD_TOOLS: ON
+            BUILD_TYPE: Debug
+            CLANG_VERSION: 11.0.0
```

Comment:
`11.0.0` is chosen because it is [the latest version](https://github.com/mapbox/mason/tree/master/scripts/clang++) available in mason.